notebooks/exploring_explorer.livemd

```diff
@@ -28,7 +28,7 @@ For CSV, your 'usual suspects' of options are available:
 * `dtypes` - A keyword list of `[column_name: dtype]`. If a type is not specified for a column, it is imputed from the first 1000 rows. (default: `[]`)
 * `header` - Does the file have a header of column names as the first row or not? (default: `true`)
 * `max_rows` - Maximum number of lines to read. (default: `nil`)
-* `null_character` - The string that should be interpreted as a nil value. (default: `"NA"`)
+* `null_character` - The string that should be interpreted as a nil value. (default: `nil`)
```

Comment:
My only concern atm is if `nil` is interpreted as `"nil"`, but this is only a documentation problem. Same comment applies to generated docs as well.

Reply:
Yeah, I agree :+1:

Reply:
I don't want to mark this as resolved, but it's probably a little bit more obvious in f9b34bba148dc710fdf6f5db024e4b54401a33de. New call-out [here](https://github.com/elixir-explorer/explorer/pull/667#discussion_r1276193406) though.

Reply:
I think the new verbiage in 8349ff32ac257ce8ebd2e7d431d9e16f388ea01f completely resolves this.